### PR TITLE
add/linter

### PR DESCRIPTION
**Pull request by j-chimienti**
changes:

```

 "scripts": {
    "format": "prettier --write \"app/**/*.js\"",
    "precommit": "lint-staged",
    "lint": "eslint app/scripts",
    "lint:fix": "eslint app/scripts --fix"
  },

```
1. add scripts to lint & format code
1. add .editorconfig
1. add pre-commit hook to format code (not linting as many errors)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,13 +1,32 @@
 {
-  "name": "classic-custom",
-  "version": "1.1.0",
-  "description": "",
+  "name": "classicetherwallet",
```
Comment (j-chimienti) [inline package.json]: add metadata

File: package.json [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
+  "bugs": {
+    "url": "https://github.com/ethereumcommonwealth/etherwallet/issues"
+  },
   "license": "ISC",
   "scripts": {
     "dev": "gulp",
-    "dist": "gulp clean"
+    "dist": "gulp clean",
+    "format": "prettier --write \"app/**/*.js\"",
+    "precommit": "lint-staged"
```
Comment (j-chimienti) [inline package.json]: run prettier and commit changes in pre-commit

File: package.json [hunk 3]
```diff
@@ -1,13 +1,32 @@
...
+  },
+  "bugs": {
+    "url": "https://github.com/ethereumcommonwealth/etherwallet/issues"
+  },
   "license": "ISC",
   "scripts": {
     "dev": "gulp",
-    "dist": "gulp clean"
+    "dist": "gulp clean",
+    "format": "prettier --write \"app/**/*.js\"",
```
Comment (j-chimienti) [inline package.json]: add format npm script

**Comment by yograterol** [comment]
@j-chimienti which linter are you using?

**Comment by j-chimienti** [comment]
I'm using https://prettier.io/ which can be run w/ `format` script. Also integrated to be run w/ pre-commit hooks

Planning on fixing conflicts shortly

File: .editorconfig [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# EditorConfig is awesome: http://EditorConfig.org
```
Comment (j-chimienti) [inline .editorconfig]: add editorconfig

File: .eslintrc.yml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+env:
```
Comment (j-chimienti) [inline .eslintrc.yml]: add eslint to improve code quality

note: not running in pre-commit hook as many errors are found (unused-var no-undef, no-dupe-keys, no-console)

**Pull request merged**